sktime/dists_kernels/__init__.py

```diff
@@ -5,6 +5,8 @@
     BasePairwiseTransformer,
     BasePairwiseTransformerPanel,
 )
+
+# todo remove these imports in 0.15.0
 from sktime.dists_kernels.compose_tab_to_panel import AggrDist, FlatDist
 from sktime.dists_kernels.dtw import DtwDist
```

Comment:
My two cents: Would it be possible to have a soft_dependency check for numba around these conflicting imports? Or push this soft_dependency check somewhere down the stack-trace

Reply:
E.g. avoid exporting these modules if numba is not installed?